mapillary_tools/uploader.py

```diff
@@ -166,6 +166,15 @@ def get_upload_file_list(import_path, skip_subfolders=False):
     return sorted(upload_file_list)
 
 
+# get a list of video files in an import path
+def get_video_file_list(import_path):
+    video_file_list = []
+    for root, dir, files in os.walk(import_path):
+        video_file_list.extend(os.path.join(root, file) for file in files if file.lower().endswith(('mp4')) and preform_upload(
+            root, file))
```

Comment:
the preform_upload function might not be applicable to video files as the logging logic is different there, why is that added? just copy paste from other listing functions?